include/SDL3/SDL_oldnames.h

```diff
@@ -860,11 +926,74 @@
 #define SDL_PixelFormatEnumToMasks SDL_PixelFormatEnumToMasks_renamed_SDL_GetMasksForPixelFormatEnum
 
 /* ##SDL_platform.h */
-#ifdef __IOS__
-#define __IPHONEOS__ __IPHONEOS___renamed___IOS__
+#ifdef SDL_PLATFORM_3DS
+#define __3DS__ __3DS___renamed_SDL_PLATFORM_3DS
+#endif
```

Comment:
I added a few for the new names, but I think the `__3DS___renamed_SDL_PLATFORM_3DS` is not good as it won't generate a compiler warning/error if it used with `#ifdef`.
I think it would be better to do something as following so it generates a preprocessor error:
```
#ifdef SDL_PLATFORM_3DS
#define __3DS__ -+__3DS___renamed_SDL_PLATFORM_3DS+-
#endif
```